go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/terraform-providers/terraform-provider-newrelic
+module github.com/prodjito/terraform-provider-newrelic
```

Comment:
Hi there, we shouldn't change this as it would point to your forked repo and break imports (tests are failing due that currently). 
```
module github.com/terraform-providers/terraform-provider-newrelic
```

Reply:
I didn't mean to include that in the PR.